### PR TITLE
feat(twirl): Adding in examples of using twirl templating. #423.

### PR DESCRIPTION
**Pull request by brbrown25**

### CODE REVIEW

**Comment by brbrown25** [comment]
This is a pretty barebones contrived example that shows off some of what you can do with twirl. Thanks for putting together such an awesome project!

File: example/src/main/scala/HelloWorldTwirl.scala [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+import zhttp.nav.Navigation
+import zhttp.service.Server
+import zio.{ExitCode, URIO}
+
+object HelloWorldTwirl extends zio.App {
+  def h2: HttpApp[Any, Nothing] = GET / "a" / *[Int] / "b" / *[Boolean] to { pathParams =>
+    val (a, b)              = pathParams.params
+    val response: UResponse = Response.text(advanced.html.index(a, b).toString())
+    response
+      .asInstanceOf[UHttpResponse]
```
Comment (tusharmath) [inline example/src/main/scala/HelloWorldTwirl.scala:13]: `asInstanceOf` is not required.

**Comment by tusharmath** [comment]
Thanks @brbrown25, I have pulled your changes in #482

File: example/src/main/scala/HelloWorldTwirl.scala (hunk 1 above)
Comment (brbrown25) [reply example/src/main/scala/HelloWorldTwirl.scala:13]: Yeah that was just copied from another test.

**Comment by brbrown25** [comment]
> Thanks @brbrown25, I have pulled your changes in #482

Thanks!

**Pull request closed without merging**